src/transports/eip_1193.rs

```diff
@@ -0,0 +1,189 @@
+//! EIP-1193 transport
+//!
+//! This transport lets you use the library inside a browser to interact with
+//! EIP-1193 providers like MetaMask. It's intended for use with Rust's
+//! WebAssembly target.
+
+#![cfg(feature = "eip-1193")]
+
+use crate::api::SubscriptionId;
+use crate::{error, DuplexTransport, Error, RequestId, Transport};
+use futures::channel::mpsc;
+use futures::future;
+use futures::future::LocalBoxFuture;
+use jsonrpc_core::types::request::{Call, MethodCall};
+use std::cell::RefCell;
+use std::collections::BTreeMap;
+use std::rc::Rc;
+use wasm_bindgen::prelude::*;
+
+/// EIP-1193 transport
+#[derive(Clone, Debug)]
+pub struct Eip1193 {
```

Comment:
To be fair I prefer this notation over one with dash or an underscore. So I'd prefer the feature and the enclosing module to be both called `eip1193` and that would align with the struct as well. I'm not super strong on this, so if the `-` notation is more correct, so be it.

Reply:
I'm following the [standardized conventions](https://doc.rust-lang.org/1.0.0/style/style/naming/README.html). I don't have a strong preference either, but given that there exists a standard I'd prefer the consistency.

Reply:
Yeah, I'm not objecting to the standard, both `eip_1993` and `eip1993` are valid module names and both adhere to the standard. It's just they come from different "source strings" `EIP-1993` and `EIP1993`. 
As I said I'm not super strong on this, if `EIP-1993` is the only correct EIP name, so bit it, I can live with `eip_1993` module name :)